app/controllers/classifier.coffee

```diff
@@ -102,6 +107,12 @@ class Classifier extends Controller
     tutorialDone = User.current?.project.tutorial_done
     doingTutorial = Subject.current?.metadata.tutorial
 
+    if User.current?
```

Comment:
Keeping track of sign-ins and sign-outs isn't really related to the classifier page. I might move this block into index.coffee, where user sign-in is detected to toggle a class on the body.